RELEASING.adoc

```diff
@@ -38,15 +46,31 @@ For GA releases this should be configured under the `central` Maven profile.
 </profile>
 ----
 
-. Ensure all Maven commands listed in `.kokoro/stage.sh` script can be executed successfully locally.
-You can add `-Dgpg.skip` in place of the actual GPG key locally.
+. Run the `prod:cloud-java-frameworks/spring-cloud-gcp/stage` Kokoro job.
+
+. In the build logs, find the ID of the staging repository. You should see a log line that looks something like this:
+```
+[INFO]  * Created staging repository with ID "comgooglecloud-1416".
+```
+The `{STAGING_ID}` in this case is `comgooglecloud-1416`.
+
+. Verify staged artifacts by going to `https://oss.sonatype.org/content/repositories/{STAGING_ID}/com/google/cloud/`
++
+NOTE: You can also view all staged artifacts at http://oss.sonatype.org.
+To get access to `com.google.cloud` group ID, please make a request similar to https://issues.sonatype.org/browse/OSSRH-52371[this one] and ask someone who already has access to this group ID to vouch for you in the issue comments.
+
+. If you want to drop the staged artifacts, run the `prod:cloud-java-frameworks/spring-cloud-gcp/drop` Kokoro job.
+Provide the staging repository ID as an environment variable (i.e. `STAGING_REPOSITORY_ID=comgooglecloud-1416`).
+
+. If you want to release the staged artifacts, run the `prod:cloud-java-frameworks/spring-cloud-gcp/promote` Kokoro job.
+Providing the staging repository ID as an environment variable (i.e. `STAGING_REPOSITORY_ID=comgooglecloud-1416`).
 
-. Invoke the Kokoro `stage` job.
-Log in to OSS Sonatype and verify the contents of the staged package.
+. Verify that the new version has been published to Maven Central by checking https://repo.maven.apache.org/maven2/com/google/cloud/spring-cloud-gcp/[here].
+This might take a while.
 
-. Invoke to the Kokoko `promote` job.
+. https://github.com/GoogleCloudPlatform/spring-cloud-gcp/releases[Create] a new release on GitHub.
 
-. Make sure that the link:https://github.com/spring-io/start.spring.io/blob/master/start-site/src/main/resources/application.yml[Spring Boot Initializr] is updated to support the new version of Spring Cloud GCP.
+. Increment the project base _patch_ version. For example, from `2.0.0-SNAPSHOT` to `2.0.1-SNAPSHOT`.
```

Comment:
Do we have a handy command for this? I remember you sharing it somewhere?

Reply:
added